sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -194,6 +196,55 @@ export class StudentService extends RecordDataModelService<Student> {
       throw error;
     }
 
+    // If sfasIndividual wasn't found, check for a partial match
+    if (!sfasIndividual) {
+      let partialMatch: SFASIndividual;
+      try {
+        partialMatch =
+          await this.sfasIndividualService.getIndividualStudentPartialMatch(
```

Comment:
The method `getIndividualStudentPartialMatch` can be combined into the `getIndividualStudent` since `getIndividualStudent` is only used for this process. In this way, the DB can answer with a single call if the student has a full or partial match.